.github/workflows/npm-publish.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         with:
           node-version: 14
       - run: npm ci
+      - uses: microsoft/playwright-github-action@v1
```

Comment:
should this go up to line 13 with the other `uses` options?

Reply:
Yea absolutely it should.  Good catch